### ISSUE TITLE
No pin on right panel map

### ISSUE THREAD

**Issue by antodippo** [xbgmsharp/piwigo-openstreetmap#64]
I'm using the latest version available here on Github, and when I select image thumbnail as pin, there's no pin at all on the right panel map. It that by choice?

**Comment by xbgmsharp** [comment]
Yes it was by design. The pin option `thumbnail` did not apply to right panel map.
However since the code has been factorize, it should be possible. So now it could be consider as a bug as it does generate an error.
```
Error: iconUrl not set in Icon options (see the docs).1 leaflet.js:7:12502
```
The SQL query does not generate the image thumbnail.
This is a good reason to move to MYSQL View which is outstanding big work.

**Comment by xbgmsharp** [comment]
A quick fix would be to replace in file picture.inc.php on line 129:
```
$js_data = array(array($lat, $lon, null, null, null, null, null, null));
```
by 
```
$js_data = osm_get_items($page);
```

https://github.com/xbgmsharp/piwigo-openstreetmap/blob/master/picture.inc.php#L129

**Comment by antodippo** [comment]
Ok, thanks a lot.

**Comment by xbgmsharp** [comment]
It is not fix. osm_get_items show all image.

**Comment by xbgmsharp** [comment]
Fix via #77
